pkgs/development/web/postman/default.nix

```
@@ -1,18 +1,53 @@
-{ stdenv, lib, gnome2, fetchurl, pkgs, xorg, makeWrapper, makeDesktopItem }:
+{ stdenv, lib, fetchurl, pkgs, makeWrapper, makeDesktopItem, wrapGAppsHook
```

pkgs no longer used, stdenv.lib can be replaced with just lib
```suggestion
{ lib, fetchurl, makeWrapper, makeDesktopItem, wrapGAppsHook
```